src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessRotors.hpp

```diff
@@ -74,6 +74,7 @@ class ActuatorEffectivenessRotors : public ModuleParams, public ActuatorEffectiv
 		RotorGeometry rotors[NUM_ROTORS_MAX];
 		int num_rotors{0};
 		bool yaw_disabled{false};
+		bool yaw_disabled_non_upwards{false}; ///< keeps yaw enabled for upward facing motors
```

Comment:
Actually the same applies for Tiltrotor, where we set the km also to 0 when yaw is controller by the tilts --> the functionality doing so shouldn't be called `enableYawControl` but `removePropellerTorque` or similar.

For later: We should discuss what we actually want to achieve with it: We want to only have the tilts reacting to control yaw, but that is currently not the case as soon as we have a non-zero tilt angle, as the thrust of the motors also has a component affecting yaw in then. Could we maybe move the tilts out of matrix 0 into matrix 1, and have the moment generated by the actuators of matrix 0 be subtracted from the torque setpoint and allocated through matrix 1? Then the motors thrust would be purely used to control roll, pitch and thrust, and the yaw by the tilts.

Sidenote: for vehicles with axes that have both effectiveness from ct and km of an actuator, like tiltrotor motors when tilted, it will get important to have the ratio of ct to km right. With the current normalized effectiveness this is otherwise not the case.